app.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const app = express();
+
+const {
+  viewCategories,
+  getMessage,
+} = require("./controllers/category.controller");
+
+const {
+  viewReviewsById,
+  notFoundErr,
+} = require("./controllers/review.controller");
+
+app.use(express.json());
```

Comment:
This should not be included yet. We only need express.json() when it comes to PATCH and POST requests, so this can be deleted until we're on that ticket.